vulture/core.py

```diff
@@ -374,22 +386,41 @@ def visit_AsyncFunctionDef(self, node):
         return self.visit_FunctionDef(node)
 
     def visit_Attribute(self, node):
+        if self.ignored(node.attr):
+            self._log('Ignoring Attribute "{}" (ignored name)'.format(
+                node.attr))
+            return
```

Comment:
Can we make this check once for all types in `Vulture._define` and fold it into the ignore check that's already there?